pymc/tests/test_distributions_timeseries.py

```diff
@@ -87,10 +87,12 @@ def test_gaussian_random_walk_init_dist_shape(init):
     grw = pm.GaussianRandomWalk.dist(mu=0, sigma=1, steps=1, init=init, size=(5,))
     assert tuple(grw.owner.inputs[-2].shape.eval()) == (5,)
 
-    grw = pm.GaussianRandomWalk.dist(mu=0, sigma=1, steps=1, init=init, shape=1)
+    # Test that if steps are none, they are inferred from shape
+    grw = pm.GaussianRandomWalk.dist(mu=0, sigma=1, steps=None, init=init, shape=1)
     assert tuple(grw.owner.inputs[-2].shape.eval()) == ()
 
-    grw = pm.GaussianRandomWalk.dist(mu=0, sigma=1, steps=1, init=init, shape=(5, 1))
+    # Test that if steps are none, they are inferred from shape
+    grw = pm.GaussianRandomWalk.dist(mu=0, sigma=1, steps=None, init=init, shape=(5, 1))
```

Comment:
These should be valid, related to ValueError check

Reply:
Theyre valid! thats why theyre in the passing test function

Reply:
O meant the original were valid, before this PR